app/helpers/orders_helper.rb

```diff
@@ -20,4 +20,8 @@ def order_detail_quantity_class(order_detail)
       "different-quantity less-quantity"
     end
   end
+
+  def show_cancel_button(order, user)
```

Comment:
This appears to be a boolean method... could you change it to `show_cancel_button?` to match Ruby convention of boolean methods?

Reply:
👍